.vscode/c_cpp_properties.json

```diff
@@ -34,6 +34,44 @@
                     "${workspaceFolder}\\src"
                 ]
             }
+        },
+        {
+            "name": "Particle_nix",
+            "includePath": [
+                "${workspaceFolder}/src",
+                "~/.particle/toolchains/deviceOS/5.3.0/user/inc",
```

Comment:
Do not include Particle

Ensure that you also include the directories specified in CMakeLists.txt